src/Item_Ticket.php

```diff
@@ -506,9 +506,9 @@ public static function showForTicket(Ticket $ticket)
                     $item->getFromDB($data["id"]);
                     echo "<td class='center'>" . $item->getKBLinks() . "</td>";
                     echo "<td class='center'>";
-                    echo Dropdown::getDropdownName("glpi_states", $data['states_id']) . "</td>";
+                    echo (isset($data["otherserial"]) ? Dropdown::getDropdownName("glpi_states", $data['states_id']) : '') . "</td>";
```

Comment:
Id do not get the relation between `otherserial` and  `states_id`.

Reply:
ping?

Reply:
Sorry, I was on holidays. This is a copy / paste error. Fixed.